src/bidmanager.js

```diff
@@ -83,6 +84,33 @@ exports.bidsBackAll = function () {
   return bidsBackAll();
 };
 
+/**
+ * If the bid is a video bid, publishers will need to be able to get the cache ID.
+ * Since the cache call is asynchronous, the getter must be too. This attaches a
+ * vastUrlCacheKey function to the bid.
+ *
+ * This function takes a callback argument, and promises to call it exactly once when the
+ * cache ID has been returned.
+ */
+function prepVideoBid(bid) {
+  // Queue callbacks which arrive before the cache has returned.
```

Comment:
This got deleted anyway... it's not necessary if we don't expose the callback to the user.